### ISSUE TITLE
FAQs - "What is the elementary schema" question answer not populated

### ISSUE THREAD

**Issue by glsdown** [elementary-data/elementary#611]
Path: /general/faq.mdx

On the FAQ page for the documentation, the "What is the Elementary Schema" doesn't appear to be populated.

https://user-images.githubusercontent.com/52132406/214065876-e35ac602-cf57-4df2-b294-dd56eb66feff.mov

**Comment by glsdown** [comment]
Additionally related to this, at the bottom of the FAQ page, it mentions:

> You could also open a [GitHub issue](https://github.com/elementary-data/elementary/issues/new), label it under documentation, and we could add the missing question (and answer) to the docs.

However, I do not have the correct permissions to add labels to issues.

**Comment by Maayan-s** [comment]
Thank you for reporting @glsdown!
The empty question seems to be an issue with a migration the docs platform we are using made. 
I contacted them and asked them to review and suggest a fix.

Meanwhile this is the content that should be there, let me know if it answers your question:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/22072028/214084200-6104f7f3-5b09-48e3-9e1a-0afc7e5b3407.png">


About the labels - you are right! 
We changed permissions in the repo and missed that. 
I just added a template for reporting docs gaps, and updated the docs ref accordingly :) 
<img width="811" alt="image" src="https://user-images.githubusercontent.com/22072028/214084827-570fe9fd-bfed-4044-a7b9-cc02099ffe3e.png">
